lib/importers/ores_scores_before_and_after_importer.rb

```diff
@@ -20,15 +20,15 @@ def initialize(articles_courses:)
   private
 
   def import_scores(wiki)
-    first_revs = []
+    all_revs = []
     last_revs = []
     @articles_courses.each do |ac|
-      first_revs << ac.all_revisions.where(wiki: wiki).order('date ASC').first
+      all_revs << ac.all_revisions.where(wiki: wiki)
       last_revs << ac.all_revisions.where(wiki: wiki).order('date ASC').last
     end
 
-    first_revs.select! { |rev| rev.present? && rev.wp10_previous.nil? }
-    RevisionScoreImporter.new(wiki.language).update_previous_wp10_scores first_revs
+    all_revs.select! { |rev| rev.present? && rev.wp10_previous.nil? }
+    RevisionScoreImporter.new(wiki.language).update_previous_wp10_scores all_revs
```

Comment:
Does this work? It looks as though with the changes above we're now adding a series of collections to `all_revs`. That is, I imagine `all_revs` is going to look like:
```
[ [ Revisions ], [ Revisions ], [ Revisions ] ]
```
This would be a different format than the `update_previous_wp10_scores` method has taken before.